src/components/config/index.ts

```diff
@@ -83,6 +85,18 @@ const handshakeAndValidate = async (config: Config) => {
   }
 }
 
+const writeConfigFile = (flags: any, parsedConfig: Config) => {
```

Comment:
can you change this function to async? so the caller can wait until the write succeeds.

Reply:
updated